app/views/oAuth/token/index.scala

```diff
@@ -70,16 +65,16 @@ object index:
               ),
               td(cls := "date")(
                 t.createdAt.map: created =>
-                  frag("Created ", momentFromNow(created), br),
+                  frag(ot.created(" "), momentFromNow(created), br),
```

Comment:
passing `" "` as the translation argument?

Should we instead have a translation for `Created: ` without arguments, then we follow it with the `momentFromNow`?

Reply:
I was passing it in order to attempt to avoid an additional github action warnings we see on translation PRs about "_... has a trailing space_". Additionally, I didn't want the space to accidentally be left out of it was part of the translation string.

I looked though some translation strings and it looks like `source/ublog.xml: safeToUseImages` is the only that actually includes a space at the end of the translation string.

How about keeping the placeholder in `Created %s` but I then do the following?
`frag(ot.created(momentFromNow(created)), br)`. I've confirmed this works (and probably how I should have just done it initially).

Reply:
I've gone ahead and applied the `ot.created(momentFromNow(created)` change where `momentFromNow` is passed in as the placeholder and this is how it appears to be done universally in the code.